hydra-gradle-core/src/main/java/com/pandora/hydra/LazyTestExcluder.java

```diff
@@ -21,12 +23,14 @@
 
     private final Supplier<HydraClient> hydraClient;
     private final String projectName;
+    private final Project project;
 
     private volatile Set<String> blacklist;
 
-    public LazyTestExcluder(String projectName, Supplier<HydraClient> hydraClientSupplier) {
+    public LazyTestExcluder(String projectName, Supplier<HydraClient> hydraClientSupplier, Project project) {
```

Comment:
We probably don't need to pass in the projectName now that the project itself is being passed in

Reply:
Yep, good point. I'll make the change.